### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mboudrig**
- Removed files we are not using
- Eslint/Prettier
- Added Examples at bottom of page
- Load readme files directly from story

### CODE REVIEW

File: src/AutoExample/components/list.js [hunk 1]
```diff
@@ -21,67 +19,66 @@ export default class List extends React.Component {
...
-          realValue: option.computed
-        };
-      });
-
+      option = option || {};
+      return {
+        id: option.id || id,
+        // value: option.label || (option.type && option.type.name) || "" + option,
+        value: option.value,
+        // realValue: isUndefined(option.value) ? option : option.value
```
[the remark this answers is not in the document]
Comment (mboudrig) [reply src/AutoExample/components/list.js]: This specific one I would like to keep, this if enum prop handlers and I have not fixed those fully yet.

File: src/StoryPage/index.js [hunk 2]
```diff
@@ -30,148 +30,128 @@ const importString = ({metadata, config, exampleImport}) =>
...
-
-  return (
-      <Tab menu={{ secondary: true, pointing: true }} panes={panes} />
-
-
-    /*
-    <TabbedView tabs={tabs(metadata)}>
-      <div className={styles.usage}>
-        
+        {componentReadme &&
```
Comment (vejandla) [inline src/StoryPage/index.js]: This is the markdown value that is being passed from the story.

File: package.json [hunk 3]
```diff
@@ -22,16 +22,28 @@
...
+    "eslint-plugin-protractor": "^1.40.0",
+    "eslint-plugin-react": "^7.10.0",
+    "eslint-plugin-standard": "^3.1.0",
     "github-markdown-css": "^2.9.0",
     "highlight.js": "^9.12.0",
     "loader-utils": "^1.1.0",
+    "prettier": "^1.11.1",
     "query-string": "^6.0.0",
-    "react-autodocs-utils": "git+ssh://git@github.com:Evolent-Health/react-autodocs-utils.git#master",
+    "react-autodocs-utils": "git+ssh://git@github.com:Evolent-Health/react-autodocs-utils.git#readme-paths",
```
Comment (evollu) [inline package.json]: shall we merge this one first?

Comment (vejandla) [reply package.json]: This was merged today.

Comment (vejandla) [reply package.json]: Can we point this back to master and merge these PR's ?

**Pull request merged**